### PR TITLE
Add to admin tasks

### PR DESCRIPTION
**Pull request by Ghalko**
## Proposal
Add manage task to admin tasks to include env variable so that testing/deploying is much simpler.

### CODE REVIEW

File: admin/tasks.py [hunk 1]
```diff
@@ -5,6 +5,21 @@
...
 
+
+@task()
+def manage(*args):
+    """Take arguments and run python manage
+
+    :param args: ex. runserver, migrate
+    """
+    if os.getcwd() != HERE:
+        os.chdir(HERE)
```
Comment (sloria) [inline admin/tasks.py]: Why is this necessary?

Comment (Ghalko) [inline admin/tasks.py]: Is that taken care of by the namespace?

Comment (sloria) [inline admin/tasks.py]: No, but why is it necessary to change the working directory in the first place?

Comment (Ghalko) [inline admin/tasks.py]: Because we need to know where manage.py is. I guess I could do a `os.path.join(HERE, '..', 'manage.py')`?

**Comment by Ghalko** [comment]
@sloria Improved, but couldn't find anything on how to handle a list of args since invoke can handle running commands sequentially on the same line.

File: tasks/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+"""Consolidates invoke tasks and utilities
+
+"""
+from .tasks import *  # noqa
```
Comment (sloria) [inline tasks/__init__.py]: I'm not sure of the side effects of having both `tasks.py` and `tasks/__init__.py`. It might make more sense to just move all the code from tasks.py into `tasks/__init__.py` to avoid potential clashes.

Comment (Ghalko) [inline tasks/__init__.py]: Is that a good approach? Is there anyplace that we import tasks.py? The only place I see a potential is in archiver/listeners.py

Comment (Ghalko) [inline tasks/__init__.py]: And is it good design practice to put code in init?

Comment (sloria) [inline tasks/__init__.py]: We wouldn't import tasks in the application code, but `invoke` must be importing the tasks code; having two conflicting modules to import `tasks.py` and `tasks/__init__.py` is hard to reason about.

It's not necessarily a bad thing to have code in `__init__.py`. For readability, we might break up `tasks/__init__.py`  into multiple files (and then import those modules in `__init__.py`), but that doesn't need to happen now.

Comment (Ghalko) [inline tasks/__init__.py]: Then I will update this shortly.

**Comment by Ghalko** [comment]
@sloria Moved, good?

**Comment by sloria** [comment]
Looks good.

Re: the `manage` command, I think what you had before--a single, required argument for the command string--was preferred. You would just need to make sure you use quotes around the command.

```python
inv manage 'runserver -p 9876'
```

That removes the need for the `target` argument, and allows any number of additional arguments to be passed to the management. commands.

**Comment by sloria** [comment]
I'll make the above change when I merge.

**Comment by Ghalko** [comment]
Sounds good. Thanks. (It will be inv admin.manage though ;)
